spec/commands/admin/update_code_group_spec.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+module Decidim
+  module AnonymousCodes
+    module Admin
+      describe UpdateCodeGroup do
+        let(:current_organization) { create(:organization) }
+        let(:current_user) { create(:user, :confirmed, :admin, organization: current_organization) }
+        let(:form_params) do
+          {
+            title: "Sample Code Group",
+            expires_at: 10.days.from_now,
+            active: true,
+            max_reuses: 10
+          }
+        end
+        let(:form) do
+          CodeGroupForm.from_params(
+            form_params
+          ).with_context(
+            current_organization: current_organization
+          )
+        end
+        let(:code_group) do
+          Decidim::AnonymousCodes::Group.create(
+            title: "Sample Code Group",
+            expires_at: 5.days.from_now,
+            active: true,
+            max_reuses: 5,
+            organization: current_organization
+          )
```

Comment:
```suggestion
    create(:anonymous_codes_group, expires_at: 5.days.from_now, active: true, max_reuses: 5, organization: organization)
```
We have a factory for it, take also into account that the title is a hash because it is translatable. If using the factory, no need to define it, you can go with the default one.

Reply:
Done